### ISSUE TITLE
iOS WebView + Dialog dim not working.

### ISSUE THREAD

**Issue by taetae98coding** [KevinnZou/compose-webview-multiplatform#34]
I just use M3 Scaffold and `androidx.compose.ui.window.Dialog` but dim not working in ios
- Android is ok
```kotlin
@Composable
internal fun WebViewScreen(
    modifier: Modifier = Modifier,
    onNavigateUp: () -> Unit,
) {
    val uiState = remember { mutableStateOf<WebViewUiState>(WebViewUiState.Url("https://www.naver.com")) }
    val webViewDialogVisibleState = remember { mutableStateOf(false) }

    Scaffold(
        modifier = modifier,
        topBar = {
            TopBar(
                onNavigateUp = onNavigateUp,
                onNavigation = { webViewDialogVisibleState.value = true },
            )
        },
    ) {
        WebView(
            modifier = Modifier.fillMaxSize()
                .padding(it),
            uiState = uiState.value,
        )
    }

    WebViewDialog(
        uiState = uiState,
        visibleState = webViewDialogVisibleState,
    )
}

@Composable
private fun WebViewDialog(
    uiState: MutableState<WebViewUiState>,
    visibleState: MutableState<Boolean>,
) {
    if (visibleState.value) {
        Dialog(
            onDismissRequest = { visibleState.value = false },
        ) {
            DialogContent(
                modifier = Modifier.fillMaxWidth(),
                uiState = uiState,
                visibleState = visibleState,
            )
        }
    }
}
```
![image](https://github.com/KevinnZou/compose-webview-multiplatform/assets/147542533/3000e6a4-f75b-4398-b45d-0383efde2041)

**Comment by KevinnZou** [comment]
Thanks for your feedback. If I understood correctly, you want to show a dialog above the WebView and it failed. However, in your screenshot, it seems that the dialog pops out successfully. Could you explain its current behaviour and your expected behaviour? Thanks!

**Comment by taetae98coding** [comment]
You can see webview and topbar.
- TopBar is dim
- But WebView not

 In Android Dialog show with dim background but ios not.

**Comment by KevinnZou** [comment]
@taetae98coding Thank you for your explanation! I also reproduced the bug. After conducting some research, I think it is the bug caused by `UIKitView`. The WebView on the iOS side is implemented by `WKWebView` wrapped inside the UIKitView. 
I tried to replace `WKWebView` with `UITextView` and the problem still exists.
![截屏2023-10-30 09 51 40](https://github.com/KevinnZou/compose-webview-multiplatform/assets/43654611/7f1ca436-3294-4866-8d09-1249ebbb49c4)


I also found a similar issue in Compose Multiplatform repo: https://github.com/JetBrains/compose-multiplatform/issues/3848. 

I will submit this bug to JetBrains and wait for their solution. Thank you for your feedback.

**Comment by KevinnZou** [comment]
@taetae98coding I have submitted the issue https://github.com/JetBrains/compose-multiplatform/issues/3881

**Comment by taetae98coding** [comment]
@KevinnZou Thank you for your relply!!

**Comment by KevinnZou** [comment]
> it is fixed compose 1.5.10

Yes, I am waiting for the release of 1.5.10.

**Comment by KevinnZou** [comment]
It has been fixed in https://github.com/KevinnZou/compose-webview-multiplatform/releases/tag/1.7.0
